src/lib/crypto/kimchi_backend/gadgets/ecdsa.ml

```diff
@@ -929,20 +918,14 @@ let%test_unit "Ecdsa.secp256k1_verify_tiny_full" =
 
             assert (Mina_stdlib.List.Length.equal base_checks.canonicals 19) ;
 
-            let base_multi_range_checks_count = ref 146 in
+            let base_range_checks_count = ref 440 in
```

Comment:
where did you get this number from? it is not 146*3 is it?

Reply:
This is the number of range checks required by the base field operations during ECDSA signature verification, not counting the additional range checks required when the curve constants `a` and/or `b` are non-zero.